frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -19,17 +19,23 @@ const partitions = [
   {
     name: "rows",
     columnFilter: isDimension,
-    title: t`Fields to use for the table rows`,
+    title: jt`Fields to use for the table ${(
+      <span className="text-dark text-heavy">rows</span>
```

Comment:
I think you have to mark "rows" for translation too.

Reply:
Ah, you're right, I got the syntax wrong. Pushed up a fix.